pkg/virt-operator/resource/generate/components/crds.go

```diff
@@ -839,9 +839,21 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *promv1.Prome
 							"description": "Container {{ $labels.container }} in pod {{ $labels.pod }} free memory is less than 20 MB and it is close to memory limit",
 							"summary":     "VM is at risk of being terminated by the runtime.",
 						},
+					},
+					{
+						Record: "num_virt_handlers_by_node_running_virt_launcher",
```

Comment:
Hi @ashleyschuett will it be possible for you to add kubevirt_ prefix to the recording rule?
There is a PR in progress to add this prefix to the other recording rules as well, https://github.com/kubevirt/kubevirt/pull/4636.

Reply:
@sradco thank you! updated